src/main/java/org/opentripplanner/apis/gtfs/mapping/routerequest/BicyclePreferencesMapper.java

```diff
@@ -0,0 +1,157 @@
+package org.opentripplanner.apis.gtfs.mapping.routerequest;
+
+import static org.opentripplanner.apis.gtfs.mapping.routerequest.ArgumentUtils.getParkingFilters;
+import static org.opentripplanner.apis.gtfs.mapping.routerequest.ArgumentUtils.getParkingPreferred;
+import static org.opentripplanner.apis.gtfs.mapping.routerequest.ArgumentUtils.parseNotFilters;
+import static org.opentripplanner.apis.gtfs.mapping.routerequest.ArgumentUtils.parseSelectFilters;
+
+import graphql.schema.DataFetchingEnvironment;
+import java.util.Set;
+import org.opentripplanner.apis.gtfs.generated.GraphQLTypes;
+import org.opentripplanner.framework.time.DurationUtils;
+import org.opentripplanner.routing.api.request.preference.BikePreferences;
+import org.opentripplanner.routing.api.request.preference.VehicleParkingPreferences;
+import org.opentripplanner.routing.api.request.preference.VehicleRentalPreferences;
+import org.opentripplanner.routing.api.request.preference.VehicleWalkingPreferences;
+
+public class BicyclePreferencesMapper {
+
+  static void setBicyclePreferences(
+    BikePreferences.Builder preferences,
+    GraphQLTypes.GraphQLBicyclePreferencesInput args,
+    DataFetchingEnvironment environment
+  ) {
+    if (args != null) {
```

Comment:
Code style, I like precondition not long if-blocks without an else:
```
if(args == null) { return; }

// main logic
```

Reply:
I've refactored the ones I quickly found in https://github.com/opentripplanner/OpenTripPlanner/pull/5185/commits/f8dc495a83a89ba628e2a899330615dc589405e2 but let me know if you find more of these.